content/en/tracing/trace_collection/admission_controller.md

```diff
@@ -31,25 +36,37 @@ Adding this annotation results in the injection of the tracer library for that l
 Valid `<language>` values are:
 - `java`
 - `js`
+- `python`
 
-For example to inject the latest Java tracer:
+For example to inject the Java tracer v1.0.0:
 
 ```yaml
 annotations:
-    admission.datadoghq.com/java-lib.version: "latest"
+    admission.datadoghq.com/java-lib.version: "1.0.0"
 ```
 
 **Note**: Use caution specifying `latest` as major library releases can introduce breaking changes.
 
 Although it's an uncommon scenario, you can add multiple `<language>-lib.version` annotations to inject multiple language tracers into one container.
 
-For example to inject the latest Java tracer and Node tracer v3.0.0:
+For example to inject the Java tracer v1.0.0, Python tracer v1.0.0, and Node tracer v3.0.0:
 ```yaml
 annotations:
-    admission.datadoghq.com/java-lib.version: "latest"
+    admission.datadoghq.com/java-lib.version: "1.0.0"
+    admission.datadoghq.com/python-lib.version: "1.0.0"
     admission.datadoghq.com/js-lib.version: "3.0.0"
 ```
 
+You can also inject the latest version of a tracer library for the Java, Python and Node language via the following annotation:
+```yaml
+annotations:
+    admission.datadoghq.com/java-lib.version: "latest"
+    admission.datadoghq.com/python-lib.version: "latest"
+    admission.datadoghq.com/js-lib.version: "latest"
+```
+
+Note that the latest keyword should be used with caution, as it may risk potential errors due to major version differences from your newly injected applications versus your existing applications that have already been deployed. 
```

Comment:
```suggestion
**Note**: Use caution specifying `latest`, as it may raise errors because of major version differences between your newly injected applications and your existing applications that have already been deployed. 
```